classes/02class/exercises/c02-network02/luizfds/c02-network02.txt

```diff
@@ -0,0 +1,32 @@
+# Subnetting
+
+Commit all your answers to a text file `c02-network02.txt` under your folder name.
+
+### Given a AWS network range, slice it in 4 small subnets and identify how many usable hosts for each one of them.
+
+- 10.0.0.0/16
+
+The network size for 10.0.0.0/16 will have 65,536 IP addresses, so:
+65,536 / 4 = 16,384 addresses per subnet
```

Comment:
This is not entirely right because when you have more subnets you will lose at least 2 ips (network and broadcast) per subnet, an extra 3 if it is on AWS infrastructure as mentioned in the classes.

Maybe just mention that the number of available hosts will depend on the number of subnets

Reply:
Hi @caiocezart,

I changed the file structure adding only what the exercise asks and now I think it's more accurated. Please let me know what you think.

Cheers,